errbot/flow.py

```diff
@@ -390,7 +391,8 @@ def execute(self, flow: Flow):
                     if syntax_args:
                         syntax += syntax_args
                     possible_next_steps.append("- %s" % syntax)
-                self._bot.send(flow.requestor, "\n".join(possible_next_steps))
+                if not flow.dismiss_prompting:
+                   self._bot.send(flow.requestor, "\n".join(possible_next_steps))
```

Comment:
PEP8 error (see Travis build output): `errbot/flow.py:395:20: E111 indentation is not a multiple of four`